### ISSUE TITLE
Conflicting rules for static and dynamic finds

### ISSUE THREAD

**Issue by stefandrissen** [Riverside-Software/sonar-openedge#781]
A static find **without** no-error:
````
find customer.
````
Triggers the findnoerror rule which warns that if no record can be found a run-time error will occur.

A dynamic find **without** no-error:
````
buffer customer:find-unique().
````
Does not trigger any rule, even though it, just like a static find, will also throw a run-time error if no record can be found. 'Normal' usage **with** no-error:
````
hb:find-unique() no-error.
````
Triggers the avoidnoerror rule. This applies to all dynamic buffer find methods.

So please sharpen the rules so that:
1. the find-unique and find-first methods should trigger the findnoerror rule
2. the find-unique and find-first methods should **not** trigger the avoidnoerror rule

**Comment by gquerret** [comment]
Will be included in next release

**Comment by cverbiest** [comment]
I tried adding 
![image](https://user-images.githubusercontent.com/1312691/158357324-7afde02a-a66f-42ea-b2e5-e5cfe84b4417.png)
to the exclude keywords but that does not help.
Can this rule be updated
